src/Illuminate/Foundation/Testing/RefreshDatabaseState.php

```diff
@@ -1,13 +0,0 @@
-<?php
```

Comment:
You shouldn't remove files in current version, some implementation can depend on it, it should target master.